tensorrt_llm/models/commanda/model.py

```
@@ -0,0 +1,204 @@
+
+        if self.mapping.is_last_pp_rank():
+            self.ln_f = LayerNorm(
+                normalized_shape=config.hidden_size,
+                eps=config.norm_epsilon,
+                bias=False,
+                dtype=config.dtype)
+
+    def forward(
+            self,
+            input_ids=None,
+            position_ids=None,
```

Ruff (0.14.6)

Unused method argument: `position_ids`

(ARG002)